INSTALL.rst

```diff
@@ -49,12 +56,30 @@ Create a virtual environment and clone the INSPIRE source code using `git`:
 
 .. code-block:: console
 
-    $ mkvirtualenv inspirehep
+    $ mkvirtualenv -p /usr/bin/python2.7 inspirehep
```

Comment:
In any case... I guess that it's safer if you don't really do strange things (like I do :S)